app/views/govuk_publishing_components/components/_summary_list.html.erb

```diff
@@ -115,7 +115,7 @@
                 <% end %>
               <% end %>
             <% else %>
-              <%= tag.dd class: "govuk-summary-list__actions" do %>
+              <%= tag.dd class: "govuk-summary-list__actions govuk-summary-list__row--no-actions" do %>
```

Comment:
Is this definitely right? The guidance says:
```
If you're not using Nunjucks macros, do not include an empty <span class="govuk-summary-list__actions"></span> in rows without actions. Instead, add the govuk-summary-list__row--no-actions modifier class to the row.```
But we weren't using an empty span in the first place - is this necessary? If it is, cool, but a bit more explanation would be good on the PR.

Reply:
I think so - does something standout?

> If you're not using Nunjucks macros, do not include an empty <span class="govuk-summary-list__actions"></span> in rows without actions.

We need the empty row

> Instead, add the `govuk-summary-list__row--no-actions` modifier class to the row.``` 

This has been done in this context

> But we weren't using an empty span in the first place - is this necessary? If it is, cool, but a bit more explanation would be good on the PR.

I can outline a bit more on the PR of why we're using `dd` instead of `span`